frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
Even though it looks like it, this expression (and the old one) does not create a watcher. Hence it doesn't matter performance-wise. But I prefer the old implementation, it's more common.